README.rst

```diff
@@ -72,6 +72,23 @@ If you don't want the errors, then make use of an extra argument to the command
 
 This last step should be performed every time you create new or make changes to existing programs.
 
+Show Programs Tab
+~~~~~~~~~~~~~~~~
+
+In order to show programs tab in the LMS dashboard, users will need to manually create an entry in ``Programs api config`` model in LMS Admin Panel. Go to ``Home > Programs >  Programs api config``. Add ``Marketing path`` equal to ``/programs`` and enable it. Then Programs tab will be shown on the LMS where users can view their registered programs. It will show like as in below picture.
```

Comment:
nit: Instead of having Home > Programs > Program api config, the admin url can make it easy for others. It is a simple step, just for ease. Not a necessity here.